src/lib/swapper/swappers/ThorchainSwapper/utils/getL1quote.ts

```diff
@@ -166,12 +163,14 @@ export const getL1quote = async (
   }
 
   const getRouteBuyAmount = (quote: ThornodeQuoteResponseSuccess) => {
-    const emitAsset = bn(quote.expected_amount_out).plus(quote.fees.outbound)
+    const emitAsset = bn(quote.expected_amount_out).plus(quote.fees.total)
```

Comment:
the amount out before fees, is the amount out after fees ( `quote.expected_amount_out` ), plus the fees `quote.fees.total`

Reply:
Quoting big kev here:

>  this change is no longer the value of emitAsset. emitAsset is expected amount out + outbound fee (https://gitlab.com/thorchain/thornode/-/blob/v1.127.1/x/thorchain/querier_quotes.go?ref_type=tags#L763). This value was being used to calculate the extra allowed slippage which improved refund rates (at least at the time)